tests/test_pipeline_files.py

```diff
@@ -146,6 +146,32 @@ def test_file_not_expired(self):
         for p in patchers:
             p.stop()
 
+    @defer.inlineCallbacks
+    def test_media_download_201(self):
+        item_url = "http://example.com/file.pdf"
+        item = _create_item_with_files(item_url)
```

Comment:
This alread creates the expected item, so the test passes even though it doesn't call the code you added.

Reply:
Could you let me know how I can mock a 201 response? This is my first time contributing.

Reply:
The problem is not about mocking a response, it's about checking that the pipeline actually worked. You are only checking that the `"file_urls"` field is correct, but that field is the input to the pipeline, not the output, it's supposed to always exist. You should check the `"files"` field like other tests do.

Reply:
I modified it to check for the checksum in "files", does it look correct now?

Reply:
Your code still doesn't run, which should have been easy to check for you. Please do this next time.